docs/zh/integration/offline_data_sources/hive.md

```diff
@@ -59,7 +59,7 @@
 
 | OpenMLDB 数据格式 | Hive 数据格式 |
 | ----------------- | ------------- |
-| BOOL              | BOOLEAN       |
+| BOOL              | BOOL       |
```

Comment:
would you add the type 'string'? we miss it too.

Reply:
> would you add the type 'string'? we miss it too.

sure